src/Symfony/Component/DependencyInjection/Definition.php

```diff
@@ -30,6 +30,8 @@ class Definition
     private $factoryMethod;
     private $factoryService;
     private $shared = true;
+    private $deprecated = false;
+    private $deprecationTemplate = 'The \'%service_id%\' service is deprecated. You should stop using it, as it will soon be removed.';
```

Comment:
I missed this! Please use double quotes, as done elsewhere (see e.g. src/Symfony/Bundle/FrameworkBundle/Controller/Controller.php)